lblod/frontend-lokaal-mandatenbeheer
# improve iv sorting

## Description

- Merge 'bcsd' service into installatievergadering. It was used for way more than just bcsd.
- track all mandataris instances in the installativergadering service and sort them
- use that sorting as a fallback if there is no rangorde to sort the mandataris instances of the other iv organs

![image](https://github.com/user-attachments/assets/97a5196e-a269-4a67-a94f-43fdb0c5b314)
(screenshot is of college vs vast bureau because in my testset that was the smallest, but gemeenteraad / rmw works the same way)
## How to test

- Go to the IV page and generate some data (or start from some test/anonymized production data)
- See that the order of the ocmw is dependent on the order of the gemeenteraad and similar for vast bureau vs college
- See that the order of the bcsd is dependent on the absolute rank of the person in other organs (mayor > schepen > voorzitter gr ... )
- See that if you change the order of e.g. gemeenteraadsleden, the order of ocmw changes immediately
